### PR TITLE
Corporate Safe Buffs

### PR DESCRIPTION
**Pull request by Geevies**
Spoilers inside!

### CODE REVIEW

**Comment by Geevies** [comment]
!review

File: code/game/objects/random/random.dm [hunk 1]
```diff
@@ -928,13 +928,13 @@
...
-	name = "random high valuable item"
-	desc = "This is a random high valuable item."
+	name = "random high value item"
+	desc = "This is a random high value item."
 	icon = 'icons/obj/coins.dmi'
 	icon_state = "coin_diamond_heads"
 	problist = list(
 		/obj/item/bluespace_crystal = 5,
-		/obj/item/stack/telecrystal{amount = 10} = 5,
+		/obj/item/stack/telecrystal{amount = 15} = 0.5,
```
Comment (Arrow768) [inline code/game/objects/random/random.dm]: That's a nogo with the current vault security level.
Antags rushing the vault is already a meme, which won't get better by increasing the tc amount in a stack by 50%

**Comment by Geevies** [comment]
Neu-Vault Back:
image

Neu-Vault Fore:
image

**Comment by Geevies** [comment]
The blast doors are linked to Command level (bridge) access.

**Comment by Geevies** [comment]
!review

**Pull request closed without merging**